plugin.cpp

```diff
@@ -165,7 +173,13 @@ string plugin_triggers(PLUGIN_HANDLE handle)
 		  it != triggers.end();
 		  ++it)
 	{
-		ret += "{ \"asset\"  : \"" + (*it).first + "\"";
+		string source = rule->getSource();
+		if (source.compare("Reading") == 0)
+			ret += "{ \"asset\"  : \"" + (*it).first + "\"";
+		if (source.compare("Statistics") == 0)
+			ret += "{ \"statistic\"  : \"" + (*it).first + "\"";
+		if (source.compare("Statistics History") == 0)
+			ret += "{ \"statisticRate\"  : \"" + (*it).first + "\"";
```

Comment:
Better to add `else if` and then in `else` block - Please catch error (Unknown data source ....)